docs/gatsby-config.js

```diff
@@ -16,29 +16,43 @@ module.exports = {
           1: 'version-1',
         },
         sidebarCategories: {
-          null: ['index', 'getting-started', 'whats-new'],
-          Essentials: [
-            'essentials/schema',
-            'essentials/server',
-            'essentials/data',
+          null: [
+            'index',
+            'getting-started',
+            'graphql-playground',
+            '[Changelog](https://github.com/apollographql/apollo-server/blob/master/CHANGELOG.md)',
           ],
-          Features: [
-            'features/caching',
-            'features/mocking',
-            'features/errors',
-            'features/data-sources',
-            'features/subscriptions',
-            'features/metrics',
-            'features/graphql-playground',
-            'features/scalars-enums',
-            'features/unions-interfaces',
-            'features/directives',
-            'features/creating-directives',
-            'features/authentication',
-            'features/testing',
-            'features/apq',
-            'features/health-checks',
-            'features/file-uploads',
+          'Defining a Schema': [
+            'schema/schema',
+            'schema/scalars-enums',
+            'schema/unions-interfaces',
+            'schema/directives',
+            'schema/creating-directives',
+          ],
+          'Serving Requests': [
+            'serving/data',
+            'serving/data-sources',
+            'serving/errors',
+            'serving/authentication',
+            'serving/file-uploads',
+            'serving/subscriptions',
+          ],
+          Testing: [
+            'testing-and-monitoring/mocking',
```

Comment:
Should we follow-up on the splitting of testing and monitoring with a decoupling of their shared path? (i.e. Changing `testing-and-monitoring` to `testing` for `mocking` and `testing`?

Reply:
Ah yeah good catches!